server/src/endpoint/claims/validate-claim.ts

```diff
@@ -3,10 +3,12 @@ import crypto, { createECDH } from "crypto";
 import { RequestHandler } from "../request-handler-wrapper";
 import { validationBadRequest } from "../../utils/errors";
 import { validate, ValidationSchema } from "../../utils/validate";
+import getConfig from "../../config";
```

Comment:
To improve testability the config is passed in as a property on the route payload.

Reply:
DI, nice

Reply:
@ahinchliff how do i use it?

```
  router.post(
    "/claims",
    requestHandlerWrapper(validateClaim, config, services)
  );
```

Reply:
https://github.com/dltxio/loda.dashboard/blob/development/server/src/routes/webhook/fireblocks.ts#L34

Reply:
Done